src/goto-programs/lazy_goto_functions_map.h

```diff
@@ -1,4 +1,4 @@
-// Copyright 2016-2017 Diffblue Limited. All Rights Reserved.
```

Comment:
There really should be a CI job that reminds people about this. Here's a script, slightly adapted from a different project:
````bash
#!/bin/bash
set -e -u -o pipefail

script=$(readlink -e "$0")
scriptdir=$(dirname "$script")

if [[ $# -gt 1 ]]
then
	echo "$script accepts at most one commit as argument" 1>&2
	exit 1
fi

if [[ $# -eq 1 ]]
then
	# specific revision requested
	commit=$(git rev-parse $1)
elif git diff HEAD --quiet --ignore-submodules=all
then
	# there are no changes in the working tree
	commit=$(git rev-parse HEAD)
else
	# check local, uncommitted modifications
	commit=
fi

if [[ -n $commit ]]
then
	# specific revision
	echo ">>> Checking commit $commit"
	year=$(git log -1 --pretty=format:%ad --date=short $commit | cut -f1 -d-)
	git_diff_cmd="git diff-tree -r $commit"
else
	# go for working tree
	echo ">>> Checking worktree"
	year=$(date +%Y)
	git_diff_cmd="git diff-index HEAD"
fi

# List all files that have been modified in the commit (or in the working tree)
# file names with spaces will cause trouble in more than one place and are
# NOT supported
function list_modified_files {
	$git_diff_cmd --no-commit-id --name-status \
		--ignore-submodules --diff-filter=ACMR \
		| (egrep '(\.cpp|\.[ch]|\.sh|Makefile)$' || true) | awk '{ print $2 }'
}

# Enumerate all years covered by copyright sequences/ranges, such as
# 2012, 2013, 2015-2017 (which would yield 2012 2013 2015 2016 2017)
function enum_years {
	local years=$1
	years=${years// /}
	ifs=$IFS
	IFS=,
	for interval in $years ; do
		if [[ $interval =~ "-" ]] ; then
			seq ${interval/-/,}
		else
			echo $interval
		fi
	done
	IFS=$ifs
}

yearpat="([[:digit:]]{4}(-[[:digit:]]{4})?,[[:space:]]+)*([[:digit:]]{4}-)?[[:digit:]]{4}"

ec=0
for f in $(list_modified_files)
do
	if ! git grep -qE \
		"Copyright[[:space:]]+$yearpat[[:space:]]+Diffblue Limited" \
		$commit -- $f; then
		echo "$f: needs copyright notice" 1>&2
		ec=1
	else
		copyright_years=$(git grep -E \
			"Copyright[[:space:]]+$yearpat[[:space:]]+Diffblue Limited" \
			$commit -- $f \
			| sed -e 's/.*Copyright[[:space:]]*//' \
				-e 's/ Diffblue Limited.*//')
		if ! enum_years "$copyright_years" | grep -wq $year
		then
			echo "$f: needs copyright update" 1>&2
			ec=1
		fi
	fi
done
exit $ec
````

Reply:
ok, sounds good. I'll suggest adding this script to our Travis installation in a separate PR.